ffv1.md

```diff
@@ -186,19 +188,19 @@ a = b, a += b, a -= b, a *= b
 
 `get_bits( i )` is the action to read the next `i` bits in the bitstream, from most significant bit to least significant bit, and to return the corresponding value. The pointer is increased by `i`.
 
-# General Description
+# Sample Encoding
```

Comment:
The specification describes the format and how to decode it. The word "encoding" is a bit confusing as it suggests that how to encode would be specified. In fact one could design a lossy encoder that outputs the same format that a lossless decoder decodes

Reply:
I found the term "General Description" to be very vague for the contents. The subsections here all seem to contextualize how samples are coded. Would `Sample Coding` work?

Reply:
Sample Coding sounds ok to me

Reply:
updated that commit to https://github.com/FFmpeg/FFV1/pull/130/commits/89e45281bc89e74dde860fb4ac7c0c0af402a346